packages/svelte-query/src/routes/+page.svelte

```diff
@@ -0,0 +1,4 @@
+<h1>Svelte Query</h1>
+<p>This routes foulder is unused</p> 
+<p>Documentation should be written in docs/svelte</p>
+<p>Examples should be added to examples/svelte</p>
```

Comment:
can we remove this if it is unused?

not really sure why we have a `lib` directory under `src`, as well as `routes`. Is this some sort of bootstrapping? Is it strictly needed?

if possible, I'd prefer to have the same structure as the other lib packages

Reply:
Turns out I could delete this without consequences! The `svelte-package` command runs within a `@sveltejs/kit` project, so it means there are some structural patterns it needs to conform to.

I _can_ set the package to build `./src` instead of `.src/lib`, but that also builds and exports the `__tests__` folder.